content/src/main/content/jcr_root/apps/acs-commons/touchui-widgets/source/touchui-composite-multifield.js

```diff
@@ -0,0 +1,303 @@
+                                                acs-aem-nested=""
+                                                name="./pages">
+                                                    <layout
+                                                    jcr:primaryType="nt:unstructured"
+                                                    sling:resourceType="granite/ui/components/foundation/layouts/fixedcolumns"
+                                                    method="absolute"/>
+                                                    <items jcr:primaryType="nt:unstructured">
+                                                        <column
+                                                        jcr:primaryType="nt:unstructured"
+                                                        sling:resourceType="granite/ui/components/foundation/container">
+                                                            <items jcr:primaryType="nt:unstructured">
+                                                                <page
+                                                                jcr:primaryType="nt:unstructured"
+                                                                sling:resourceType="granite/ui/components/foundation/form/textfield"
+                                                                fieldDescription="Name of Page"
+                                                                fieldLabel="Page Name"
+                                                                name="./page"/>
+                                                                <path
+                                                                jcr:primaryType="nt:unstructured"
+                                                                sling:resourceType="granite/ui/components/foundation/form/pathbrowser"
+                                                                fieldDescription="Select Path"
+                                                                fieldLabel="Path"
+                                                                name="./path"
+                                                                rootPath="/content"/>
+                                                            </items>
+                                                        </column>
+                                                    </items>
+                                                </field>
+                                            </pages>
+                                        </items>
+                                    </column>
+                                </items>
+                            </fieldset>
+                        </items>
+                    </column>
+                </items>
+            </content>
+        </jcr:root>
+     </code>
+*/
+
+
+(function () {
+    var DATA_ACS_AEM_NESTED = "data-acs-aem-nested",
```

Comment:
sure, makes sense, and btw i am going to implement all review comments at one go, eod